### DISCUSSION TITLE
Side door to configure IP address on Sling Solo?

### DISCUSSION THREAD

**Discussion by as400jockey** [GerryDazoo/Slinger#277] [Q&A] [answered]
I got my two M1 boxes to work.  This is awesome.

I finally found my Solo box.  Finally figured out that I'd hard coded an IP address on the thing.  It's not doing DHCP.  I now have two subnets - 192.168.1.x (home) and 192.168.0.x (business).  Previously only had the 192.168.0.x network.

Configured ip is 192.168.0.94 . I found my notes with the admin password and the port.  If I fire up Slinger on the 192.168.0.x network I can connect to the solo box.  So it's working.  But I need it on my 192.168.1.x network with all my other stuff.

What I really need to do is to reconfigure the fixed IP on this box to go back to using DHCP.

Loaded up slingplayer desktop v2.x.x.x (the one that talks to Solo and original Slingbox).  Fires up OK, finds the solo, plays the stream. But I can't config because it's trying to Phone Home and nobody answers.

I tried http on port 80, 8080, 8800.... tried telnet.  Tried ssh.  Tried FTP.

Anything that I try to Google comes up with Sling's "have a nice life" page.

Anyone know a side door to get into this Sling Solo so that I can do some reconfig?  Don't want to do a factory reset as then I'd be totally screwed.

**Comment by jkaponis** [comment, marked as answer]
if you use the slingplayer 1.5 you can change the ip and port.
http://www.rmrsoft.com/slingplayer15.exe

**Comment by as400jockey** [comment]
Outstanding!

I got it reconfigured and now it’s working beautifully.

Many thanks.

**Reply by robosmith** [reply]
Better block your Solo's access to the Internet before Mr. Sling trashes it.

**Reply by sp222222** [reply]
yeah I was thinking the same thing...probably too late now

**Comment by GerryDazoo** [comment]
I've been recommending that people do NOT run any Sling supplied software. From reports, I believe that SP1.5 did in fact do a firmware update and bricked a box.
I'm sorry I didn't see this post earlier the "right" thing to do would have been to setup a second interface on your windows box to give it a presence on the  192.168.1.x  subnet. Or change the subnet mask on 192.168.0.x to also include the 192.168.1.x subnet  (255.255.254.0)

**Reply by Mikoshi-nyudo** [reply]
> I've been recommending that people do NOT run any Sling supplied software. From reports, I believe that SP1.5 did in fact do a firmware update and bricked a box. 

If you're referring to sp22222's solo, it wasn't the SP 1.5 that did that btw. It was due to the Firewall rules that weren't setup properly, unfortunately, and the box could still communicate to Sling's servers. How do I know? because I helped him secure it properly after the fact. If it was another setup, it would be prudent to ensure that it wasn't something else similar. However, I concur that one must not have internet connected, using Sling's setup assistant, unless you really know what you're doing. Just important to be accurate or sure of the circumstances, not intending to step on other's toes or views. Regards.

**Reply by sp222222** [reply]
> I've been recommending that people do NOT run any Sling supplied software. From reports, I believe that SP1.5 did in fact do a firmware update and bricked a box. I'm sorry I didn't see this post earlier the "right" thing to do would have been to setup a second interface on your windows box to give it a presence on the 192.168.1.x subnet. Or change the subnet mask on 192.168.0.x to also include the 192.168.1.x subnet (255.255.254.0)

Gerry, if I'm the report, I must say that I did not have my router properly blocking things to AND from sling servers....
I will test my blocking rules tomorrow (again 3rd time) with an old AV I still have that works with 1.5 spm to see if I can do things to it.

**Reply by sp222222** [reply]
> > I've been recommending that people do NOT run any Sling supplied software. From reports, I believe that SP1.5 did in fact do a firmware update and bricked a box.
> 
> If you're referring to sp22222's solo, it wasn't the SP 1.5 that did that btw. It was due to the Firewall rules that weren't setup properly, unfortunately, and the box could still communicate to Sling's servers. How do I know? because I helped him secure it properly after the fact. If it was another setup, it would be prudent to ensure that it wasn't something else similar. However, I concur that one must not have internet connected, using Sling's setup assistant, unless you really know what you're doing. Just important to be accurate or sure of the circumstances, not intending to step on other's toes or views. Regards.

I cannot thank you enough this morning for getting me straighten up. I'm ashamed to make a living as a desktop PC support specialist and not knowing the proper way to lock down a router for outbound connections, which is what the slingbox itself does to get the killswitch firmware.  

I owe you a case of beer, big time.

**Reply by Mikoshi-nyudo** [reply]
> I owe you a case of beer, big time.

Nah, you're already passing it on and helping others. That's enough thanks for me.

**Comment by as400jockey** [comment]
Thanks for that. I was able to avoid any other setup options and go straight to the specification of the IP address and netmask. It seems to have worked out for me. I was using the oldest version of 1.5 that I could find.

In my case, changing the net mask wasn't an option. I need to keep those two networks separated to try and isolate as much Network traffic on the home side or the business side of the network. The only work around I could have done would be to add another router between the two networks on what had been the default gateway address when I set up the solo all those many years ago.

I will suggest to others that if you go into that software that you choose the options to manually configure your network and manually configure your router. Perhaps I got lucky and avoided what you are warning about. But so far so good.

The solo is currently pointing at a dummy address for default gateway, so it should not be able to communicate with the interweb. As for the m1s, I blocked their IP addresses at the firewall, so they should be avoiding any communication to the outside world as well. I did notice that UPnP had the two ports open on the firewall. Now that I'm blocking those two ips, I'm going to see if UPnP closes those ports. I could not manually close them. And I couldn't block those ports because UPnP had told the router to keep them open.



Sent from my Verizon, Samsung Galaxy smartphone



-------- Original message --------
From: GerryDazoo ***@***.***>
Date: 11/12/22 5:15 PM (GMT-05:00)
To: GerryDazoo/Slinger ***@***.***>
Cc: "David C. Shea" ***@***.***>, Author ***@***.***>
Subject: Re: [GerryDazoo/Slinger] Side door to configure IP address on Sling Solo? (Discussion #277)


I've been recommending that people do NOT run any Sling supplied software. From reports, I believe that SP1.5 did in fact do a firmware update and bricked a box.
I'm sorry I didn't see this post earlier the "right" thing to do would have been to setup a second interface on your windows box to give it a presence on the 192.168.1.x subnet. Or change the subnet mask on 192.168.0.x to also include the 192.168.1.x subnet (255.255.254.0)

-

**Reply by sp222222** [reply]
if you close the UPnP won't rebooting the router (or esp powering it off for 2-3mins) close those open ports?
